### PR TITLE
Added more details in Readme 

### PR DESCRIPTION
**Pull request by deepak1725**
**Added:**
* Demo gif 
* Working Flow
* Github Repo Info

In respect to https://issues.redhat.com/browse/APPAI-1136

### CODE REVIEW

**Comment by deepak1725** [comment]
@SriKrishnaPaparaju All Changes in the closest context of https://marketplace.visualstudio.com/items?itemName=redhat.fabric8-analytics. are done.

**Comment by deepak1725** [comment]
@jeffmaury Can you please review

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -6,24 +6,63 @@
...
+        </li>
+      </ul>
+      <p>
+        <img src="https://raw.githubusercontent.com/redhat-developer/intellij-dependency-analytics/master/src/main/resources/images/demo.gif" alt="Demo">
+      </p>
+
+      <h2 id="user-content-know-more-about-dependency-analytics-platform" class="link-header">Know more about Dependency Analytics Platform</h2>
+      <p>
+        The mission of this project is to significantly enhance developer experience:
+        providing Insights(security, licenses, AI based guidance) for applications and helping developers, Enterprises.
```
Comment (jeffmaury) [inline src/main/resources/META-INF/plugin.xml]: Shouldn't we have a space after Insights  and before ( ?

File: src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -6,24 +6,63 @@
...
+      </ul>
+
+      <h2 id="user-content-feedback-%26-questions" class="link-header">Feedback &amp; Questions</h2>
+      <ul>
+        <li>File a bug in <a href="https://github.com/redhat-developer/intellij-dependency-analytics/issues" rel="noopener noreferrer" target="_blank">GitHub Issues</a></li>
+      </ul>
+
+      <h2 id="user-content-license" class="link-header">License</h2>
+      <p>
+        Apache 2.0, See <a href="https://github.com/redhat-developer/intellij-dependency-analytics/blob/master/LICENSE" rel="noopener noreferrer" target="_blank">LICENSE</a>
```
Comment (jeffmaury) [inline src/main/resources/META-INF/plugin.xml]: The project license is EPL-2.0

File: src/main/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -6,24 +6,63 @@
...
       </ul>
+
+      <h2>Features</h2>
+      <ul>
+        <li>Opening or editing a manifest file (<code>pom.xml</code> / <code>package.json</code> / <code>requirements.txt</code>) scans your application for security vulnerabilities,
+            flag them along with 'quick fixes'.
+        </li>
+      </ul>
+      <p>
+        <img src="https://raw.githubusercontent.com/redhat-developer/intellij-dependency-analytics/master/src/main/resources/images/demo.gif" width=700 alt="Demo">
```
Comment (jeffmaury) [inline src/main/resources/META-INF/plugin.xml]: What is the rationale for having width=700 when the image is 916 pixels large ?

Comment (deepak1725) [reply src/main/resources/META-INF/plugin.xml]: @jeffmaury fixed.

**Comment by deepak1725** [comment]
@jeffmaury can you please approve.?

**Pull request merged**